src/Test.FunctionalTests.Sarif/Multitool/ValidateCommandTests.cs

```diff
@@ -215,6 +215,10 @@ public void SARIF2010_ProvideCodeSnippets_Valid()
         public void SARIF2010_ProvideCodeSnippets_Invalid()
             => RunInvalidTestForRule(RuleId.ProvideCodeSnippets);
 
+        [Fact]
+        public void SARIF2010_ProvideCodeSnippets_WithArtifacts_Valid()
+            => RunTest("SARIF2010.ProvideCodeSnippets_WithArtifacts_Valid.sarif");
```

Comment:
>WithArtifacts [](start = 54, length = 13)

Let's move this just below the _other_ "valid" test for this rule. Also, it's not the "with artifacts" that's important here, it's that the artifacts have embedded file content. So perhaps `"SARIF2010.ProvideCodeSnippets_WithEmbeddedContent.sarif"`.